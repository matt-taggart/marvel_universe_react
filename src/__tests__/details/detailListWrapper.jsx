import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import DetailListWrapper from '../../components/details/detailListWrapper';

Enzyme.configure({ adapter: new Adapter() })

describe('Detail List Wrapper', () => {
  it('Should render correctly', () => {
    const wrapper = shallow(
      <DetailListWrapper heading="Spiderman">
        <div>Hello world!</div>
      </DetailListWrapper>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
