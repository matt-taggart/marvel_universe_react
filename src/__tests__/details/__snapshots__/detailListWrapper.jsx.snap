// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detail List Wrapper Should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <DetailListWrapper
    heading="Spiderman"
  >
    <div>
      Hello world!
    </div>
  </DetailListWrapper>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="media-content"
      >
        <div
          className="content"
        >
          <div>
            <strong>
              Spiderman
            </strong>
            <br />
            <div>
              Hello world!
            </div>
          </div>
        </div>
      </div>,
      "className": "media",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="content"
        >
          <div>
            <strong>
              Spiderman
            </strong>
            <br />
            <div>
              Hello world!
            </div>
          </div>
        </div>,
        "className": "media-content",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div>
            <strong>
              Spiderman
            </strong>
            <br />
            <div>
              Hello world!
            </div>
          </div>,
          "className": "content",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <strong>
                Spiderman
              </strong>,
              <br />,
              <div>
                Hello world!
              </div>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Spiderman",
              },
              "ref": null,
              "rendered": "Spiderman",
              "type": "strong",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Hello world!",
              },
              "ref": null,
              "rendered": "Hello world!",
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
    "type": "article",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="media-content"
        >
          <div
            className="content"
          >
            <div>
              <strong>
                Spiderman
              </strong>
              <br />
              <div>
                Hello world!
              </div>
            </div>
          </div>
        </div>,
        "className": "media",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="content"
          >
            <div>
              <strong>
                Spiderman
              </strong>
              <br />
              <div>
                Hello world!
              </div>
            </div>
          </div>,
          "className": "media-content",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div>
              <strong>
                Spiderman
              </strong>
              <br />
              <div>
                Hello world!
              </div>
            </div>,
            "className": "content",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <strong>
                  Spiderman
                </strong>,
                <br />,
                <div>
                  Hello world!
                </div>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Spiderman",
                },
                "ref": null,
                "rendered": "Spiderman",
                "type": "strong",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Hello world!",
                },
                "ref": null,
                "rendered": "Hello world!",
                "type": "div",
              },
            ],
            "type": "div",
          },
          "type": "div",
        },
        "type": "div",
      },
      "type": "article",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
